src/Tasks/WriteCodeFragment.cs

```diff
@@ -170,9 +185,15 @@ private string GenerateCode(out string extension)
             }
 
             // For convenience, bring in the namespaces, where many assembly attributes lie
-            foreach (string name in NamespaceImports)
+            // This will be skipped for only a single situation:
+            // - When Language is "C#" and sets ImplicitUsings to "enable".
+            // Checking only ImplicitUsings is unknown if it works for Visual Basic.
+            if (ImplicitUsings is "disable" or "false")
             {
-                globalNamespace.Imports.Add(new CodeNamespaceImport(name));
+                foreach (string name in NamespaceImports)
+                {
+                    globalNamespace.Imports.Add(new CodeNamespaceImport(name));
+                }
             }
 
             foreach (ITaskItem attributeItem in AssemblyAttributes)
```

Comment:
The code will need to figure out a way to filter the fully qualified attributes and shorten them up to just their types when ``ImplicitUsings`` is ``enable`` or ``true``.